openhtf/exe/phase_executor.py

```diff
@@ -69,15 +81,15 @@ class PhaseOutcome(collections.namedtuple('PhaseOutcome', 'phase_result')):
   phase_result is an instance of Exception, then that is the Exception that
   was raised by the phase.  The raised_exception attribute can be used as
   a convenience to test for that condition, and the is_timeout attribute can
-  similarly be used to check for the timeout case.
+  similarly be used to check for the timeout case. 
 
   The only accepted values for phase_result are None (timeout), an instance
   of Exception (phase raised), or an instance of openhtf.PhaseResult.  Any
   other value will raise an InvalidPhaseResultError.
   """
   def __init__(self, phase_result):
     if (phase_result is not None and
-        not isinstance(phase_result, (Exception, openhtf.PhaseResult))):
+        not isinstance(phase_result, (ExceptionOutcome, openhtf.PhaseResult))):
```

Comment:
this is missing some lines, you should rebase on station_api, as that's about to get merged into master any day now.